third-party/realdds/include/realdds/dds-stream-profile.h

```diff
@@ -15,7 +15,7 @@ namespace realdds {
 
 union dds_stream_uid
 {
-    uint32_t whole;
+    uint32_t whole = 0;

```

Comment:
Redundant
If a non-static data member has a [default member initializer](https://en.cppreference.com/w/cpp/language/data_members#Member_initialization) and also appears in a member initializer list, then the member initializer is used and the default member initializer is ignored
[Link](https://en.cppreference.com/w/cpp/language/constructor)

Reply:
Hehe
And the default ctor? Better to show it.

Reply:
Good point. It was hidden in a collapsed part of the code.

But seeing the two added initializations made me wonder which initialization prevails :-)